components/TextFields/src/ContainedInputView/MDCBaseTextFieldLayout.h

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MDCBaseTextFieldLayout : NSObject
+
+@property(nonatomic, assign) BOOL leftViewHidden;
+@property(nonatomic, assign) BOOL rightViewHidden;
+
+@property(nonatomic, assign) CGRect leftViewFrame;
+@property(nonatomic, assign) CGRect rightViewFrame;
+
+- (instancetype)initWithTextFieldSize:(CGSize)textFieldSize
+                             leftView:(UIView *)leftView
+                         leftViewMode:(UITextFieldViewMode)leftViewMode
+                            rightView:(UIView *)rightView
+                        rightViewMode:(UITextFieldViewMode)rightViewMode
+                                isRTL:(BOOL)isRTL
+                            isEditing:(BOOL)isEditing;
+
+@end
+
+NS_ASSUME_NONNULL_END
```

Comment:
Please use explicit nullability annotations throughout this file. Our code base does not make use of NS_ASSUME_NONNULL_BEGIN.

Reply:
Whoops sorry, forgot to remove it!